.github/workflows/release-edc.yml

```diff
@@ -0,0 +1,76 @@
+name: Create EDC Release
+on:
+  workflow_dispatch:
+    inputs:
+      edc_version:
+        description: 'Version string that is used for publishing. Appending -SNAPSHOT will create a snapshot release.'
+        required: true
+        type: string
+
+
+env:
+  EDC_VERSION: ${{ github.event.inputs.edc_version || inputs.edc_version }}
+
+jobs:
+  Prepare-Release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: releases
+
+      # create tag on the current branch using GitHub's own API
+      - name: Create tag on current branch (main)
```

Comment:
This confuses me, since at line 20 the selected branch seems to be "releases", not "main"

Reply:
correct. fixed.